ballerina/tests/schemaClient-test.bal

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/sql;
+
+@test:Config {
+    groups: ["schemaClientTest"]
+}
+function testListTablesWorking() returns error? {
+    SchemaClient client1 = check new("localhost", "root", "password", "testDB");
+    string [] tableList = check client1->listTables();
+    check client1.close();
+    test:assertEquals(tableList, ["employees","offices"]);
+}
+
+@test:Config {
+    groups: ["schemaClientTest"]
+}
+function testListTablesFail() returns error? {
+    SchemaClient client1 = check new("localhost", "root", "password", "testDB1");
+    string [] tableList = check client1->listTables();
+    check client1.close();
+    test:assertEquals(tableList, []);
+}
+
+@test:Config {
+    groups: ["schemaClientTest"]
+}
+function testGetTableInfoNoColumns() returns error? {
+    SchemaClient client1 = check new("localhost", "root", "password", "testDB");
+    sql:TableDefinition 'table = check client1->getTableInfo("employees", include = sql:NO_COLUMNS);
+    check client1.close();
+    test:assertEquals('table, {"name":"employees","type":"BASE TABLE"});
+}
+
+@test:Config {
+    groups: ["schemaClientTest"]
+}
+function testGetTableInfoColumnsOnly() returns error? {
+    SchemaClient client1 = check new("localhost", "root", "password", "testDB");
+    sql:TableDefinition 'table = check client1->getTableInfo("employees", include = sql:COLUMNS_ONLY);
+    check client1.close();
+    test:assertEquals('table.name, "employees");
+    test:assertEquals('table.'type, "BASE TABLE");
+    test:assertEquals((<sql:ColumnDefinition[]>'table.columns).length(), 8);  
```

Comment:
Double check whether it is possible to check the order

Reply:
Need to determine whether the columnar information is fetched correctly.

Reply:
Better to check for everything
https://github.com/ballerina-platform/module-ballerina-sql/blob/master/ballerina/tests/metadata-test.bal#L59